chapter6/euna/chapter6-2.java

```diff
@@ -19,12 +19,18 @@ public class Geometry {
     public double area(Object shape) throws NoSuchShapeException {
         if (shape instanceof Square) {
             /* 구현하세요 */
+            Square square = (Square)shape;
+            return square.side * square.topLeft;
```

Comment:
topLeft는 좌측 상단 좌표라고 생각해서 저는 아래처럼 작성했어요!
```suggestion
            return square.side * square.side;
```